### ISSUE TITLE
Update libfprint, fprintd

### ISSUE THREAD

**Issue by worldofpeace** [NixOS/nixpkgs#68876]
**Describe the bug**
I might some back to this when I have the time.

**Packages and versions to upgrade**
[fprintd v0.9.0](https://gitlab.freedesktop.org/libfprint/fprintd/-/tags/V_0_9_0)
[libfprint v1.0](https://gitlab.freedesktop.org/libfprint/libfprint/-/tags/V_1_0)

**Comment by Vskilet** [comment]
Hi,
I try to build the master-unstable and I have a problem when fprintd is build. It seems that `automake` isn't in dependencies :

```
/build/fprintd-0.9.0/missing: line 81: automake-1.16: command not found
WARNING: 'automake-1.16' is missing on your system.
         You should only need it if you modified 'Makefile.am' or
         'configure.ac' or m4 files included by 'configure.ac'.
         The 'automake' program is part of the GNU Automake package:
         <http://www.gnu.org/software/automake>
         It also requires GNU Autoconf, GNU m4 and Perl in order to run:
         <http://www.gnu.org/software/autoconf>
         <http://www.gnu.org/software/m4/>
         <http://www.perl.org/>
make[1]: *** [Makefile:352: Makefile.in] Error 127
make[1]: Leaving directory '/build/fprintd-0.9.0/data'
make: *** [Makefile:451: check-recursive] Error 1
```

**Comment by worldofpeace** [comment]
Fixed in ca065d92fd39bf5a4a98a91b07ee0e31c4d02a2a.
